scipy/stats/_stats_py.py

```diff
@@ -854,6 +867,9 @@ def tstd(a, limits=None, inclusive=(True, True), axis=0, ddof=1):
     return np.sqrt(tvar(a, limits, inclusive, axis, ddof))
```

Comment:
I think you can call `tvar` with `_no_deco`. I think this will reduce some overhead.

Reply:
Ah, yes, done!